testsuite/libra-fuzzer/src/lib.rs

```diff
@@ -12,7 +13,12 @@ pub mod fuzz_targets;
 /// Implementation for a particular target of a fuzz operation.
 pub trait FuzzTargetImpl: Sync + Send + fmt::Debug {
     /// The name of the fuzz target.
-    fn name(&self) -> &'static str;
+    fn name(&self) -> &'static str {
+        module_path!()
+            .rsplit("::")
+            .next()
+            .expect("module path must have at least one component")
+    }
```

Comment:
You may want to double check this but this may not be what you want. [`module_path`](https://doc.rust-lang.org/stable/std/macro.module_path.html) expands based on the context of where the macro physically is located so unless this default impl is overridden, I believe all impls will have module_path expanded to the root module.

Reply:
Here's an example in the [playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=ef151b3e3647417c81f675c6fc760a09)